pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -48,6 +48,16 @@ const (
 
 	// defaultConfig is the default service config (that does not correspond to an actual service)
 	defaultConfig = "default"
+
+	// policyCheck is the node label holding the policy check option override
+	policyCheck = "POLICY_CHECK"
+)
+
+// Policy check options are disable, enable (fail-closed), and fail-open (check but not apply).
+const (
+	PolicyCheckEnable   string = "enable"
+	PolicyCheckDisable         = "disable"
+	PolicyCheckFailOpen        = "fail-open"
```

Comment:
back to my aside, I think these three values are non-obvious. i would expect to be able to use "fail-closed" as well as "enable". i don't know how much we care about consistency with the mesh-wide options, but I'd be fine with:

```
policy.sidecar.istio.io/check: {disable, enable-allow-on-error, enable-deny-on-error}
```

Thoughts?

Reply:
Sounds reasonable.

What about client vs server-side checks and protocols? Do we want extra dimensions?

We may want to force allow-on-error client policy checks for HTTP only, but not anything else?

This is getting confusing, so maybe @ozevren or someone from config can ship what's the right way to go about the overrides?

Reply:
Yeah, that is sorta why I was asking about long-term approaches in the related bug. This feels like something that should be designed with the future of config in mind.